src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -119,28 +102,29 @@ Props) => {
             </div>
           }
         >
-          <NavLink to={membersPageRoute}>
-            <Heading
-              appearance={{ size: 'normal', weight: 'bold' }}
-              text={MSG.title}
-              textValues={{
+          <ClickableHeading
+            linkTo={membersPageRoute}
+            appearance={{ margin: 'none' }}
+          >
+            <FormattedMessage
+              {...MSG.title}
+              values={{
                 count: colonyWatchers?.length,
                 hasCounter,
                 isContributorsSubsection,
               }}
             />
-          </NavLink>
+          </ClickableHeading>
         </Tooltip>
-        {/* {!isContributorsSubsection && (
+        {!isContributorsSubsection && (
           <InviteLinkButton
-            colonyName={colonyName}
+            colonyName={name}
             buttonAppearance={{ theme: 'blueWithBackground' }}
           />
-        )} */}
+        )}
       </div>
     ),
-    // [members, membersPageRoute, isContributorsSubsection, colonyName],
-    [isContributorsSubsection, membersPageRoute, colonyWatchers],
+    [isContributorsSubsection, membersPageRoute, colonyWatchers, name],
   );
 
   if (!colonyWatchers) {
```

Comment:
This hasn't been committed by you but I don't think this check works:
```
const colonyWatchers = useMemo(() => watchers?.items || [], [watchers]);
```
`colonyWatchers` can be either an array of watchers or an empty array, neither of those will ever make `!colonyWatchers` true. Please have a look into this

Reply:
good catch, will fix this